core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/RegionPlanGenerator.scala

```diff
@@ -84,23 +84,27 @@ abstract class RegionPlanGenerator(
       .toSet
   }
 
-  def populateDownstreamLinks(
+  def populateDependeeLinks(
```

Comment:
this method returns a DAG but the returned value is not being used by the caller. Does it change the DAG in-place or returns a copy? Also please add scala-doc of this method.

Reply:
Changed return type and added scala-doc.